netflowv5/searchlibrary/Most Common Port Flows.query

```diff
@@ -4,4 +4,4 @@ stats count mean(Duration) as AvgDurationNS sum(Bytes) as TotalBytes sum(Pkts) a
 limit 20 |
 lookup -r network_services [Protocol Port] [proto_number service_port] (service_name service_desc) |
 lookup -r ip_protocols Protocol Number Name as ProtocolName |
-table Port service_name count TotalBytes TotalPackets Protocol ProtocolName AvgDurationNS service_desc
\ No newline at end of file
+table Port service_name count TotalBytes TotalPackets ProtocolName ProtocolName AvgDurationNS service_desc
```

Comment:
What is this trying to do? This previously tabled both Protocol and ProtocolName (the enriched value from lookup), and not just tables ProtocolName twice?